docs/guides/kubernetes/getting-started-with-load-balancing-on-a-lke-cluster/index.md

```diff
@@ -146,6 +146,7 @@ The Linode CCM accepts annotations that configure the behavior and settings of y
 | `check-attempts` | &bull; integer <br> &bull; value between `1`-`30` | None | Number of health checks to perform before removing a back-end Node from service. |
 | `check-passive` | Boolean | `false` | When `true`, `5xx` status codes will cause the health check to fail. |
 | `preserve` | Boolean | `false` | When `true`, deleting a LoadBalancer service does not delete the underlying NodeBalancer |
+| `nodebalancer-id` | integer | None | When set, service uses existing NodeBalancer with nodebalancer-id |
```

Comment:
Annotation appears [here](https://github.com/linode/linode-cloud-controller-manager/blob/eaadb3fb7250e13dd659ce952541a09459b67bdd/cloud/linode/loadbalancers.go#L46). I was able to use it to connect service to existing nodebalancer.